SwanCustomEnvironments/swancustomenvironments/scripts/builders/accpy.sh

```diff
@@ -33,7 +33,13 @@ eval "${ACTIVATE_ENV_CMD}"
 
 # Install packages in the environment and the same ipykernel that the Jupyter server uses
 _log "Installing packages from ${REQ_PATH}..."
-pip install -r "${REQ_PATH}" "ipykernel==${IPYKERNEL_VERSION}" ${NXCALS} 2>&1
+if [ "${RESOLVED_REQ}" = true ]; then
+    # Use the same pip configuration as the Acc-Py default pip
+    ACCPY_PIP_CONF="-i $(pip config get global.index-url) --allow-insecure-host $(pip config get global.trusted-host)"
+    uv pip install ${ACCPY_PIP_CONF} -r "${REQ_PATH}" "ipykernel==${IPYKERNEL_VERSION}" ${NXCALS} 2>&1
```

Comment:
There is a problem here, we are including `nxcals` but if the user did not specify it in the requirements.txt, we would be fully resolving it with uv. The same goes for `ipykernel`.